public/js/shopping-list-item.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+class ShoppingListItem {
+  constructor(name, description) {
+    this.name = name;
+    this.description = description;
+    this.isDone = false;
+  }
+
+  check() {
+    this.isDone = true;
+  }
+
+  unCheck() {
+    this.isDone = false;
+  }
+
+  render() {
+    return `<li class="completed_${this.isDone}"><span class="grocery">${
```

Comment:
Since you're using a string template literal, it's okay to add in newlines to make this easier to read.